doc/mgr/ansible.rst

```diff
@@ -41,25 +41,46 @@ Enable the Ansible orchestrator module and use it with the :ref:`CLI <orchestrat
 Configuration
 =============
 
+The external Ansible Runner Service uses TLS mutual authentication to allow clients to use the API.
+A client certificate and a key files should be provided by the Administrator of the Ansible Runner Service for each manager node.
+This files should be copied in each of the manager nodes with read access for the ceph user.
+The destination folder for this files and the name of the files must be the same always in all the manager nodes,
+althought the certificate/key content of this files logically will be different in each node.
+
 Configuration must be set once the module is enabled by first time.
 
 This can be done in one monitor node via the configuration key facility on a
 cluster-wide level (so they apply to all manager instances) as follows::
 
+In first place, configure the Ansible Runner Service client certificate an key:
+
+    If the client certificate provided is for several servers (generic) use:
+    # ceph ansible set-ssl-certificate -i <location_of_the_crt_file>
+    # ceph ansible set-ssl-certificate-key -i <location_of_the_key_file>
 
-    # ceph config set mgr mgr/ansible/server_addr <ip_address/server_name>
-    # ceph config set mgr mgr/ansible/server_port <port>
-    # ceph config set mgr mgr/ansible/username <username>
-    # ceph config set mgr mgr/ansible/password <password>
-    # ceph config set mgr mgr/ansible/verify_server <verify_server_value>
+    If the client certificate provided is for an especific manager server use:
+    # ceph ansible set-ssl-certificate <server> -i <location_of_the_crt_file>
+    # ceph ansible set-ssl-certificate-key <server> -i <location_of_the_key_file>
+
+After setting the client certificate & key files to use, finish configuration as follows:
+    # ceph config set mgr mgr/ansible/server_location <ip_address/server_name>:<port>
+    # ceph config set mgr mgr/ansible/verify_server <False|True>
+    # ceph config set mgr mgr/ansible/ca_bundle <path_to_ca_bundle_file>
 
 Where:
 
     * <ip_address/server_name>: Is the ip address/hostname of the server where the Ansible Runner Service is available.
     * <port>: The port number where the Ansible Runner Service is listening
-    * <username>: The username of one authorized user in the Ansible Runner Service
-    * <password>: The password of the authorized user.
-    * <verify_server_value>: Either a boolean, in which case it controls whether the server's TLS certificate is verified, or a string, in which case it must be a path to a CA bundle to use in the verification. Defaults to ``True``.
+    * <verify_server_value>: boolean, it controls whether the Ansible Runner Service server's TLS certificate is verified. Defaults to ``True``.
+    * <path_to_ca_bundle_file>: Path to a CA bundle to use in the verification.
+
+In order to check that everything is OK, use the "status" orchestrator command.
```

Comment:
can you make this a link to the orchestrator_cli.rst command?

Reply:
I haven't put the link here ... i wanted just to show clearly how to get the curent status of the orchestrator. I have added the link in the Operations section